packages/lab/src/file-drop-zone/FileDropZoneIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import { StatusIndicator, ValidationStatus } from "@salt-ds/core";
+import { forwardRef } from "react";
+import { IconProps, UploadIcon } from "@salt-ds/icons";
+
+export interface FileDropZoneIconProps extends IconProps {
+  /**
+   * Status indicator to be displayed.
+   */
+  status?: ValidationStatus;
+}
+
+export const FileDropZoneIcon = forwardRef<
+  SVGSVGElement,
+  FileDropZoneIconProps
+>(function FileDropZoneIcon({ status, size = 2, ...rest }, ref) {
+  return (
+    <div>
+      {status ? (
+        <StatusIndicator ref={ref} status={status} size={size} {...rest} />
```

Comment:
Couple questions here, does this mean the UploadIcon size is fine by default?
props are only applied when there's a status and the ref is only valid when there's a status